modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCachePutAllRequest.java

```diff
@@ -17,17 +17,17 @@
 
 package org.apache.ignite.internal.processors.platform.client.cache;
 
+import java.util.LinkedHashMap;
+import java.util.Map;
 import org.apache.ignite.internal.binary.BinaryRawReaderEx;
 import org.apache.ignite.internal.processors.platform.client.ClientConnectionContext;
 import org.apache.ignite.internal.processors.platform.client.ClientResponse;
-
-import java.util.LinkedHashMap;
-import java.util.Map;
```

Comment:
Here and in other places, imports should not be re-ordered. See this page for details: https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-PackageImporting

Reply:
It was wrong formated before this patch: `Import should be ordered alphabetically, but packages starting with 'java.' should be declared prior to others.` 
Now it exactly matches coding guidelines.